.github/workflows/dependabot-auto-approve-minor.yml

```
@@ -24,6 +24,7 @@ jobs:
           - ioredis
           - fastify
           - node
+          - p-retry
```

```suggestion
          - p-
```

All of these should be safe to approve